core/src/main/resources/error/error-classes.json

```
@@ -276,6 +276,11 @@
     ],
     "sqlState" : "22008"
   },
+  "DECIMAL_PRECISION_EXCEEDS_MAX_PRECISION" : {
+    "message" : [
+      "Decimal precision <precision> exceeds max precision <maxPrecision>"
```

minor but for consistency:
```suggestion
      "Decimal precision <precision> exceeds max precision <maxPrecision>."
```